modules/ai/assets/js/editor/components/prompt-library-link.js

```diff
@@ -0,0 +1,20 @@
+import { Typography, Link } from '@elementor/ui';
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+const PromptLibraryLink = ( props ) => {
+	return (
+		<Typography variant="body2" color="text.secondary">
+			{ __( 'For more inspiration, try experimenting with proven prompts from our ' ) }
+			<Link href={ props.libraryLink } className="elementor-clickable">
+				{ __( 'prompt library' ) }
```

Comment:
Is there a need for a space before?
```suggestion
				{ ' ' } { __( 'prompt library' ) }
```